packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -90,6 +93,17 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    // Python dependencies installed
+    if (hasPythonSixModule) {
+      pythonStatus = ValidationType.installed;
+    } else {
+      pythonStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'Python installation missing module "six".\n'
+        'Install via \'sudo easy_install six\'.'
```

Comment:
Updated to suggest pip or easy_install (in that order). I couldn't think of a concise way of suggesting that they switch back to the system default and TBH, it'd be nice not to force people away from their preferences, so leaving at that :)